src/components/ExploreNav/styled.js

```diff
@@ -0,0 +1,66 @@
+import styled from "styled-components";
+import { BLACK, H3, CREAM, PINK, PointerStyles } from "../../styles/styles";
+import { min } from "../../styles/breakpoints";
+import { AnchorLink } from "gatsby-plugin-anchor-links";
+
+export const StyledLink = styled.div`
+  white-space: nowrap;
+  height: 3rem;
+  width: 100%;
+  transform: skew(15deg);
+  background-color: transparent;
+  ${PointerStyles}
+  border: 2px solid ${CREAM};
+  text-align: center;
+  &:hover {
+    background-color: ${PINK};
+  }
+`;
+
+export const LeftLink = styled(StyledLink)`
+  border-right: none;
+`;
+
+export const CenterLink = styled(StyledLink)`
+  border-right: none;
+  border-left: none;
+`;
+
+export const RightLink = styled(StyledLink)`
+  border-left: none;
+`;
+
+export const LinkContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  position: sticky;
+  top: 0;
+  z-index: 1;
+  margin-left: 5%;
```

Comment:
nit: if you change this to `margin: auto` you don't need to update it across breakpoints!!